packaging/rpm/SPECS/nvidia-container-toolkit.spec

```diff
@@ -62,6 +64,7 @@ fi
 %files
 %license LICENSE
 %{_bindir}/nvidia-container-runtime-hook
+%{_bindir}/nvidia-cdi-hook
```

Comment:
I think this should be droped from this section and added to the `%files base` section below where `%{_bindir}/nvidia-ctk` is also defined.

Reply:
Good point. I should have done that to begin with. Done.